website/src/components/CustomerLogos/index.tsx

```diff
@@ -4,85 +4,93 @@ import Marquee from "react-fast-marquee";
-            width={70}
-            height={70}
-            className="opacity-75 md:opacity-25 mx-auto"
-          />
-        </Link>
+    <section
+      className="flex w-full h-full max-w-full max-h-full items-center justify-center m-0 p-0 list-none opacity-100 overflow-hidden"
+      style={{
+        maskImage:
+          "linear-gradient(to right, rgba(0, 0, 0, 0) 0%, rgb(0, 0, 0) 35%, rgb(0, 0, 0) 65%, rgba(0, 0, 0, 0) 100%)",
+      }}
+    >
+      <div className="mx-8 grid grid-cols-3 md:grid-cols-7 gap-8">
+        <ul className="content-center">
+          <Link href="https://corrdyn.com">
+            <Image
+              alt="corrdyn logo"
+              src="/images/logos/cust-logo-corrdyn-gray.svg"
+              width={125}
+              height={125}
+              className=" mx-auto"
+            />
+          </Link>
+        </ul>
+        <ul className="content-center">
+          <Link href="https://square1.io">
+            <Image
+              alt="square1 logo"
+              src="/images/logos/cust-logo-square1-gray.svg"
+              width={100}
+              height={100}
+              className=" mx-auto"
+            />
+          </Link>
+        </ul>
+        <ul className="content-center">
+          <Link href="https://bunq.com">
+            <Image
+              alt="bunq logo"
+              src="/images/logos/cust-logo-bunq-gray.svg"
+              width={100}
+              height={100}
+              className=" mx-auto"
+            />
+          </Link>
+        </ul>
+        <ul className="content-center">
+          <Link href="https://wolfram.com">
+            <Image
+              alt="wolfram logo"
+              src="/images/logos/cust-logo-wolfram-gray.svg"
+              width={75}
+              height={75}
+              className=" mx-auto"
+            />
+          </Link>
+        </ul>
+        <ul className="content-center">
+          <Link href="https://sebgroup.com">
+            <Image
+              alt="seb logo"
+              src="/images/logos/cust-logo-seb-gray.svg"
+              width={75}
+              height={75}
+              className=" mx-auto"
+            />
+          </Link>
+        </ul>
+        <ul className="content-center">
+          <Link href="https://teracloud.com">
+            <Image
+              alt="teracloud logo"
+              src="/images/logos/cust-logo-teracloud-gray.svg"
+              width={125}
+              height={125}
+              className="  mx-auto"
+            />
+          </Link>
+        </ul>
+        <ul className="content-center col-start-2 md:col-auto">
+          <Link href="https://double11.co.uk">
+            <Image
+              alt="double11 logo"
+              src="/images/logos/cust-logo-double11-gray.svg"
+              width={70}
+              height={70}
+              className=" mx-auto"
+            />
+          </Link>
+        </ul>
       </div>
-    </div>
+    </section>
```

Comment:
I think we want to keep this as a `</div>`

```suggestion
    </div>
```